pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/index.tsx

```diff
@@ -57,25 +57,23 @@ export const DatabaseDetailsPageV2 = () => {
     },
   ];
 
-  const dbName =
-    error?.status === 404 || !data
-      ? "Database Not Found"
-      : data.metadata.dbName;
+  const dbName = isLoading ? (
+    <Skeleton paragraph={false} title={{ width: 100 }} />
+  ) : (
+    data?.metadata.dbName ?? "Database Not Found"
+  );
 
   const breadCrumbItems = [
     { name: "Databases", link: DB_PAGE_PATH },
     {
-      name: dbName,
-      link: null,
+      name: <>Database: {dbName}</>,
+      link: "",
```

Comment:
whats the difference between `null` and `""` for a link?

Reply:
I was having issues with the typing of the prop passed which expected this to be a string value and not null. Providing the empty string will still remoce the link from the breadcrumb item.